linera-views/src/lib.rs

```diff
@@ -70,6 +70,9 @@ pub mod batch;
 /// The definitions used for the `KeyValueStore` and `Context`.
 pub mod common;
 
+/// The generation of the random namespaces.
+pub mod namespace;
```

Comment:
Are we missing `#[cfg(with_testing)]` ?

Reply:
So, yes, I reverted this change and now the test memory are protected by `[cfg(with_testing)]`.